android/lib/src/main/java/com/swmansion/gesturehandler/TapGestureHandler.kt

```diff
@@ -156,7 +156,7 @@ class TapGestureHandler : GestureHandler<TapGestureHandler>() {
   companion object {
     private const val MAX_VALUE_IGNORE = Float.MIN_VALUE
     private const val DEFAULT_MAX_DURATION_MS: Long = 500
-    private const val DEFAULT_MAX_DELAY_MS: Long = 500
+    private const val DEFAULT_MAX_DELAY_MS: Long = 200
```

Comment:
https://github.com/software-mansion/react-native-gesture-handler/pull/1609

copied fix for taps not working on ios